src/App/Action/Backend/Order/AbstractOrderAction.php

```diff
@@ -7,24 +7,34 @@
 use InvalidArgumentException;
 use MyParcelNL\Pdk\App\Action\Contract\ActionInterface;
 use MyParcelNL\Pdk\App\Order\Collection\PdkOrderCollection;
+use MyParcelNL\Pdk\App\Order\Contract\OrderStatusServiceInterface;
 use MyParcelNL\Pdk\App\Order\Contract\PdkOrderRepositoryInterface;
 use MyParcelNL\Pdk\App\Order\Model\PdkOrder;
 use MyParcelNL\Pdk\Base\Support\Utils;
 use Symfony\Component\HttpFoundation\Request;
 
 abstract class AbstractOrderAction implements ActionInterface
 {
+    /**
+     * @var \MyParcelNL\Pdk\App\Order\Contract\OrderStatusServiceInterface
+     */
+    protected $orderStatusService;
```

Comment:
niet hier doen, nu hebben alle ongerelateerde actions ook deze injectie terwijl dat helemaal niet nodig is. Alleen injecten in de actie die dit gaat gebruiken